lib/stripe/object_types.rb

```diff
@@ -86,13 +89,15 @@ def self.object_names_to_classes
         Refund.object_name => Refund,
         Reporting::ReportRun.object_name => Reporting::ReportRun,
         Reporting::ReportType.object_name => Reporting::ReportType,
+        ReserveTransaction.object_name => ReserveTransaction,
         Reversal.object_name => Reversal,
         Review.object_name => Review,
         SetupAttempt.object_name => SetupAttempt,
         SetupIntent.object_name => SetupIntent,
         ShippingRate.object_name => ShippingRate,
         Sigma::ScheduledQueryRun.object_name => Sigma::ScheduledQueryRun,
         Source.object_name => Source,
+        SourceMandateNotification.object_name => SourceMandateNotification,
```

Comment:
Can we avoid this one. Sources have been deprecated for years and we're finally moving forward with a real deprecation date. Seems like a waste to add to remove later.

Additionally: most of those resources are "hacks" that `source` on `BalanceTransaction` can expand to. Only `Application` is a real one. So if I had a magic wand I'd just suppress all of those for now.